app/src/main/res/values/colors.xml

```diff
@@ -87,7 +87,7 @@
     <color name="contrast_text_dark_theme">@color/primary_text_dark_theme</color>
     <color name="caption_text_dark_theme">@color/photonLightGrey70</color>
     <color name="foundation_dark_theme">#1C1B22</color>
-    <color name="inset_dark_theme">#32313C</color>
+    <color name="inset_dark_theme">#52525E</color>
```

Comment:
We have a colour variable for this colour. Can you use @color/photonDarkGrey10?

I think you also do a search and replace where the other instance of #52525E and replace it with the colour variable as well.

Reply:
@gabrielluong Replaced all instances of #52525E with @color/photonDarkGrey10 👍